matrix-meetings-widget/src/components/meetings/ScheduleMeetingModal/ScheduleMeeting.test.tsx

```diff
@@ -873,15 +873,16 @@ describe('<ScheduleMeeting>', () => {
   it('should disabled submission if meeting series is changed to start in the past', async () => {
```

Comment:
sorry to nitpick but "should disable" and "should not disable" is the more correct - in this line and the others below